server/views/pages/record-attendance/activities.njk

```diff
@@ -9,109 +10,128 @@
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-three-quarters">
-            <h1 class="govuk-heading-l govuk-!-margin-bottom-3">{{ activities.length }} {{ 'activity' if activities.length == 1 else 'activities' }} scheduled for {{ date | formatDate('do MMMM yyyy') + ' (' + slot + ')' }}</h1>
+            <h1 class="govuk-heading-l govuk-!-margin-bottom-3">{{ activities.length }} {{ 'activity' if activities.length == 1 else 'activities' }} scheduled for {{ date | formatDate('do MMMM yyyy', true)}}</h1>
             <ul class="govuk-list">
                 <li><a href="select-period" class="govuk-link govuk-!-font-size-19">View the schedule for a different day</a></li>
             </ul>
         </div>
     </div>
+
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-            {% set rows = [] %}
-            {% for activity in activities %}
-                {% set rows = (rows.push([
-                    {
-                        attributes: {
-                            id: 'activity-' + loop.index,
-                            "data-sort-value": activity.name
-                        },
-                        html: '
-                                <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
-                                    <a href="activities/' + activity.id + '/attendance-list" class="govuk-link govuk-link--no-visited-state">' + activity.name + '</a>
-                                </h2>
-                              '
-                    },
-                    {
-                        attributes: {
-                            id: 'location-' + loop.index,
-                            "data-sort-value": activity.location
+        <div class="govuk-grid-column-two-thirds">
+            <form>
+                <input type="hidden" name="date" value="{{ date | formatDate('yyyy-MM-dd') }}">
+                {{ searchBar({
+                    inputParams: {
+                        id: 'searchTerm',
+                        name: 'searchTerm',
+                        label: {
+                            text: 'Search activities by name',
```

Comment:
Minor - text does not match that of the ticket.  In the ticket 'Search by activity name'